hnn_core/network.py

```diff
@@ -330,7 +330,7 @@ def _update_cells(self):
                     cell.gid = self.gid_ranges[cell_type][cell_idx]
                     cell.pos = pos
                     cells.append(cell)
-                    self.cells[cell_type] = cells
+                self.cells[cell_type] = cells
```

Comment:
Subtle, assuming this is a silent fix where the final behavior is the same?

Reply:
Correct. When I was trying to find the root of the discrepany between the two parallel backends, I realized that we don't need to assign the list of cell objects to the `net.cells` dict every time it iterates through the  innermost loop. Just a super minor performance improvement.